File: 0520-detect-capital/0520-detect-capital.js
/**
 * @param {string} word
 * @return {boolean}
 */
var detectCapitalUse = function(word) {
    var cap=0;
    for(c in word){
        if (word[c]>='A' && word[c]<='Z')cap+=1;
    }
    return (cap==0 || cap == word.length || (cap==1 && word[0]>='A' && word[0]<='Z'))
};

File: 0520-detect-capital/0520-detect-capital.cpp
class Solution {
public:
    bool detectCapitalUse(string word) {
        int cnt=0;
        for(auto t : word){
            if(isupper(t))cnt++;
        }
        return (cnt==word.size() || !cnt || (cnt==1 && isupper(word[0])));
    }
};